apps/judicial-system/web/src/components/Tags/TagCaseState/TagCaseState.tsx

```diff
@@ -1,38 +1,35 @@
-import { FC } from 'react'
+import { FC, useContext } from 'react'
 import { IntlShape, useIntl } from 'react-intl'
 
-import { Tag, TagVariant } from '@island.is/island-ui/core'
+import { Tag as CaseStateTag, TagVariant } from '@island.is/island-ui/core'
 import {
+  isDistrictCourtUser,
   isIndictmentCase,
   isInvestigationCase,
   isSuccessfulServiceStatus,
 } from '@island.is/judicial-system/types'
 import {
-  CaseIndictmentRulingDecision,
+  CaseListEntry,
   CaseState,
-  CaseType,
   Defendant,
   IndictmentDecision,
   User,
 } from '@island.is/judicial-system-web/src/graphql/schema'
 
+import { UserContext } from '../../UserProvider/UserProvider'
 import { strings } from './TagCaseState.strings'
 
+interface CaseStateTag {
```

Comment:
This interface has the same name as an import. Can that cause confusion?

Reply:
Don't know why we were importing the Tag component as `CaseStateTag` 🙈 Fixed..